table/table.go

```diff
@@ -67,6 +67,8 @@ var (
 	ErrInvalidRecordKey = terror.ClassTable.New(codeInvalidRecordKey, "invalid record key")
 	// ErrTruncateWrongValue returns for truncate wrong value for field.
 	ErrTruncateWrongValue = terror.ClassTable.New(codeTruncateWrongValue, "Incorrect value")
+	// ErrTrgInvalidCreationCtx happens when inserting a value outside the table partitions.
+	ErrTrgInvalidCreationCtx = terror.ClassTable.New(codeTrgInvalidCreationCtx, "locate partition failed")
```

Comment:
s/locate/Locate?

Reply:
As you see, both i and I is used here.
I think the message is not important.